src/components/Filters/RangeInput.tsx

```diff
@@ -202,14 +202,14 @@ export function RangeInput(props: RangeInputProps): JSX.Element | null {
         {renderInput(minRangeInput, handleMinChange, 'Min')}
         <div className='w-2.5 text-sm text-neutral'>-</div>
         {renderInput(maxRangeInput, handleMaxChange, 'Max')}
-      </div>
-      {isDisabled &&
+        {isDisabled &&
         <div className={cssClasses.tooltipContainer}>
```

Comment:
I would have thought eslint would want this indented, but I guess not?

Reply:
nope, I guess this is technically a sibling node of the div above so it works either way if you want it indented or not